commands/bump.js

```diff
@@ -21,15 +21,17 @@ module.exports = {
         let player = await client.Manager.get(message.guild.id);
         if (!player) return client.sendTime(message.channel, "❌ | **Nothing is playing right now...**");
         
-		// check if args[0] - 1 is a valid number between 2 and player.queue.length - 1
+		// Check if (args[0] - 1) is a valid index
 		let trackNum = parseInt(args[0] - 1);
         if (trackNum < 1 || trackNum > player.queue.length - 1) {
 			return client.sendTime(message.channel, "❌ | **Invalid track number.**");
-		}
+        }
         
-        // Swap array elements
-        [player.queue[0], player.queue[trackNum]] = [player.queue[trackNum], player.queue[0]];
-		client.sendTime(message.channel, "✅ | **" + player.queue[0].title + "** has been moved to the front of the queue.");
+        // Remove from and shift array
+        const track = player.queue[trackNum];
+        player.queue.splice(trackNum, 1);
+        player.queue.unshift(track);
+		client.sendTime(message.channel, "✅ | **" + track.title + "** has been moved to the front of the queue.");
```

Comment:
/root/sudhan/poppin-fresh/commands/bump.js:34
                client.sendTime(message.channel, "✅ | **" + track.title + "** has been moved to the front of the queue.");
                                                                  ^
TypeError: Cannot read property 'title' of undefined
    at Object.run (/root/sudhan/poppin-fresh/commands/bump.js:34:53)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)